feature/session/src/main/res/values/strings.xml

```diff
@@ -17,6 +17,7 @@
     <string name="session_detail_resource">Resources</string>
     <string name="session_detail_resource_video">Video</string>
     <string name="session_detail_resource_slide">Slides</string>
+    <string name="session_detail_share_url">https://droidkaigi.jp/2019/en/timetable/%s</string>
```

Comment:
if you add the `translatable="false"` parameter to this, then the warning will go away 😉 
``` xml
    <string name="session_detail_share_url" translatable="false">https://droidkaigi.jp/2019/en/timetable/%s</string>
```